api_server.py

```
@@ -0,0 +1,1193 @@
+                response_data = {
+                    "results": formatted_results,
+                    "total": len(formatted_results),
+                    "source": "database"
+                }
+                return jsonify(response_data) if return_json else response_data
+            except Exception as e:
+                logger.warning(f"Database query failed: {str(e)}")
+                # Fall through to JSON files
+        
+        # Try to get from JSON files
+        json_files = get_json_files()
```

Ruff (0.11.9)

Undefined name `get_json_files`

(F821)